packages/calendar/src/utils/stateless/time/range/set-range.ts

```diff
@@ -29,11 +29,11 @@ const getRangeEndGivenDayBoundaries = (
     calendarConfig.dayBoundaries.value.end
   )
   let newRangeEndDate = toDateString(date)
-  if (calendarConfig.isHybridDay) {
-    newRangeEndDate = addDays(newRangeEndDate, 1)
-  }
+  //if (calendarConfig.isHybridDay) {
```

Comment:
Do we need to change the range for this feature? This would be quite a big breaking change, which I can't just introduce like that.

If it's about the event being filtered out due to the range, we should probably look into changing the filtering instead. Is that why you made the adjustments in this file?